src/common/scripts/services.js

```diff
@@ -0,0 +1,31 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('user_management.common');
+
+    module.service('catchErrors', [
+        function () {
+            this.all = function (response, fields) {
```

Comment:
@wytrych yes new library / module should be open source. 

It would probably be easiest to make  the initial version of the new module / functionality as compatible / similar as possible with the existing `fieldErrors` functionality, this will simplify the migration path and make it possible to track any changes easily in the new module.